doc/plugins/eu.esdihumboldt.hale.doc.xtraserver/html/introduction/intro.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<script type="text/javascript"
+	src="PLUGINS_ROOT/org.eclipse.help/livehelp.js">
+</script>
+<title>Introduction to the XtraServer plugin for <i>hale studio</i></title>
+</head>
+<body>
+	<h1>XtraServer plugin for <i>hale studio</i></h1>
+
+	<p>
+	XtraServer is a product of interactive instruments. It is a suite of implementations of various OGC service specifications, e.g. Web Feature Service (WFS) and Web Map Service (WMS).
+	</p>
+	<p>
+	XtraServer services can base on any application schema according to the Geography Markup Language (GML). For this, a mapping from the GML application schema to the table structure of the underlying database is to be provided in the configuration of the service.
+	</p>
+	<p>
+	The mapping language of XtraServer is very flexible and can virtually map all GML application schemas to heavily deviating database schemas. For this reason, mappings can be quite complex.
+	</p>
+	
+	<p>
+	The purpose of this plugin is to
+	<ul>
+		<li>transform the XtraServer mappings to hale alignments (via Import) and</li>
+		<li>generate new XtraServer mapping file easily (via Export)</li>
+	</ul>
+	</p>
+	
+	<h2>Installation requirements</h2>
+	
+	The plug-in is already bundled in the standard hale studio distribution, starting from version X.X.X. 
```

Comment:
If we can resolve the open issues soon we can include it in the 3.4.0 release. But we can also adapt that after the merge.